src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -54,6 +66,40 @@ function LHNOptionsList({
         onFirstItemRendered();
     }, [onFirstItemRendered]);
 
+    const renderEmptyLHNSubtitle = useCallback(
```

Comment:
I think that `useMemo` is more appropriate than `useCallback` here and will simplify things